app/assets/javascripts/cookies.js

```diff
@@ -0,0 +1,3 @@
+var tz = jstz.determine();
+Cookies.set("timezone", tz.name());
```

Comment:
how is setting a timezone cookie helping ?

use jquery to do it ? `$.cookie("test", 1);
`